components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/Nimbus.kt

```diff
@@ -156,6 +156,26 @@ interface NimbusInterface {
      */
     fun resetTelemetryIdentifiers() = Unit
 
+    /**
+     * Records the `exposure` event in telemetry.
+     *
+     * This is a manual function to accomplish the same purpose as passing `true` as the
+     * `recordExposureEvent` property of the `getVariables` function. It is intended to be used
+     * when requesting feature variables must occur at a different time than the actual user's
+     * exposure to the feature within the app.
+     *
+     * In the case where the use of this function is required, then the `getVariables` function
+     * should be called with `false` so that the exposure event is not recorded when the variables
+     * are fetched.
```

Comment:
This PR doesn't seem to call this method from `getVariables`. We should probably fix that before landing.

Reply:
Maybe a merge snafu, I will investigate.